src/Components/Image/index.js

```diff
@@ -7,47 +7,60 @@ import {
   fullHeight,
 } from './style.module.css';
 
-const Image = ({
-  src,
-  alt,
-  caption = false,
-  fullHeight: isFullHeight = false,
-  maxHeight = 'auto'
-}) => {
-  const style = {
-    maxHeight,
-  };
-
-  if (maxHeight !== 'auto' && isFullHeight === false) {
-    style.width = 'auto';
+class Image extends React.Component {
```

Comment:
I think Hirad ended up making a version of this with hooks. Might have been added after you branched off of `artist-pages`?

Reply:
Ahh ok, i'll poke around for it and merge it in.

Reply:
All looks like it is in here: #24